Tests/LibRegex/Regex.cpp

```diff
@@ -458,10 +458,12 @@ TEST_CASE(a_star)
     String haystack = "[Window]\nOpacity=255\nAudibleBeep=0\n";
     EXPECT_EQ(re.search(haystack.view(), result, PosixFlags::Multiline), true);
     EXPECT_EQ(result.count, 32u);
```

Comment:
I think we do stuff like this without error checking all the time, as long as the out-of-bound access `VERIFY`s, I think this should be fine, right? The `EXPECT_EQ` is in my opinion like a `VERIFY` here, meaning have now checked for the  case where `result.count != 32u`. If we introduce this for all tests, they will just get unnecessarily complex and hard to read, I think.

Reply:
It stops the tests from progressing further, which is fine if you think of it as just a CI check, but very annoying if you're trying to debug it.

Reply:
I'm gonna leave the commit as-is, though tbh I always thought `EXPECT_EQ_FORCE()` did this ("expect equal or bail"), it does some other thing though.